.travis.yml

```diff
@@ -9,4 +9,17 @@ services:
 before_install:
   - bundle install
 
-script: bundle exec kitchen verify
+
+env:
```

Comment:
Are you sure you need this? I think the matrix is controlled by suiteXplatform in Test Kitchen.